packages/kmod-5.15-nvidia/kmod-5.15-nvidia.spec

```diff
@@ -59,6 +60,15 @@ Requires: %{name}-tesla(fabricmanager)
 %description fabricmanager
 %{summary}.
 
+%package open-gpu-%{tesla_major}
+Summary: NVIDIA %{tesla_major} Open GPU driver
+Version: %{tesla_ver}
+License: MIT OR GPL-2.0-only
```

Comment:
You're sure this is "OR" and not "AND"?

That's also what I gathered from this sentence, though it's a bit ambiguous:
> However, when linked together to form a Linux kernel module, the resulting Linux kernel module is dual licensed as MIT/GPLv2.

Reply:
I believe it is "OR" based upon the wording.